src/utils/formHelperFunctions.js

```diff
@@ -367,7 +368,7 @@ async function setFormElementError({
  * @param inlineErrors
  */
 function getInlineError(field, inlineErrors) {
-  const data = inlineErrors[field.servar.key];
+  const data = inlineErrors[field?.servar?.key ?? field?.id];
```

Comment:
took me a little bit to figure out that this was the last missing piece

Reply:
oops yeah we kinda overload the definition of "id". imo maybe field.servar? field.servar.key : field.id

so we can get rid of the really broad elvis operators